docker-assets/auth-api/Gemfile

```diff
@@ -0,0 +1,5 @@
+
```

Comment:
leading blank line?

Reply:
LOLvim :trollface:

Reply:
I think something happened here and this is resolved.

Reply:
yeah, it was a leading blank line, and git re-resolved this somewhere else.